experiment_1/scripts/FlexMex1_10/FlexMex1_10_preprocessing.py

```diff
@@ -33,145 +34,12 @@
 
 scalars = pd.read_csv(os.path.join(experiment_paths['data_raw'], 'Scalars.csv'), header=0)
 
+# Prepare oemof.tabular input CSV files
 create_default_elements_files(os.path.join(data_preprocessed_path, 'elements'))
```

Comment:
These could move to main() at the end of the file for better overview.

Reply:
Resolved by https://github.com/modex-flexmex/oemo-flex/pull/20/commits/fb83b24a4bc2d63f5c288b856250d26530e8baa3